api/gulpfile.js

```diff
@@ -176,6 +187,3 @@ gulp.task('undeploy', gulp.series(
   'delete_cloudwatch_logs'
 ));
 
-gulp.task('bootstrap', gulp.series('deploy','generate_sample_users', 'generate_sample_data'));
-
```

Comment:
I would also keep this default task which allows the simple use of just typing "gulp" to run the project.